listenbrainz/webserver/views/feedback_api.py

```diff
@@ -315,4 +315,18 @@ def import_feedback():
         counts = lastfm.import_feedback(user["id"], data["user_name"])
         return jsonify(counts)
 
+    if data["service"] == "navidrome":
+        if not all(k in data for k in ("navidrome_url", "auth_token", "salt", "user_name")):
+             raise APIBadRequest("Missing required Navidrome credentials (navidrome_url, auth_token, salt, user_name)")
+        
+        from listenbrainz.domain import navidrome
+        counts = navidrome.import_starred_tracks(
```

Comment:
We'll need to think more about how to implement this. As in the current form, it won't support importing from local instances that are not exposed on public internet. We could implement it on the frontend but has its own issues and is more prone to breaking. 

Will need to discuss with others when they are back from holidays.